SubmissionTest.java

```diff
@@ -148,15 +148,21 @@ public void testZip() throws IOException
         
         // Get actual Submission filenames.
         File actualDir = new File("zips");
-        File[] actualZips = actualDir.listFiles();
-        Arrays.sort(actualZips);
-        String[] actual = new String[actualZips.length];
-        for (int i = 0; i < actual.length; i++)
+        
+        // Make sure directory has not been deleted 
+        // by deleteZips test or application. 
+        if (actualDir.exists())
         {
-            actual[i] = actualZips[i].getName();
-        }
+            File[] actualZips = actualDir.listFiles();
+            Arrays.sort(actualZips);
+            String[] actual = new String[actualZips.length];
+            for (int i = 0; i < actual.length; i++)
+            {
+                actual[i] = actualZips[i].getName();
+            }
 
-        assertArrayEquals("should be same", expected, actual);
+            assertArrayEquals("should be same", expected, actual);
```

Comment:
@KaraBeason No I think that's probably the better way to go. Or - what do you think about removing the deleteZips altogether for now? Putting that in the backlog to maybe be implemented when we deal with the whole application implementation - like making it work on the various OSs?